tools/convert_characters/convert_characters.py

```diff
@@ -1,41 +1,50 @@
 #!/usr/bin/env python
-#By, Guruprasad Ananda.
+"""
+By, Guruprasad Ananda.
+"""
+from __future__ import print_function
+
+import re
+import sys
 
-import sys, re
 
 def stop_err(msg):
     sys.stderr.write(msg)
     sys.exit()
```

Comment:
It would be good to remove `stop_err` from all these scripts, which did not exit with an error code and it's just overhead.

```suggestion
```